services/apps/portal/src/providers/typeorm/migrations/1673042070216-update_app_statuses.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class updateAppStatuses1673042070216 implements MigrationInterface {
+  name = 'updateAppStatuses1673042070216';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `UPDATE "portal"."application_status" SET "label" = 'Submitted to ALC' WHERE "code" = 'SUBM'`,
```

Comment:
I wonder if this executed successfully without ';'

Reply:
I believe so, only need ; if you are using multiple statements, if its just a single statement Postgres doesn't seem to care